main/ZsensorHTU21.ino

```diff
@@ -0,0 +1,124 @@
+/*  
+  OpenMQTTGateway Addon  - ESP8266 or Arduino program for home automation 
+ 
+   Act as a wifi or ethernet gateway between your 433mhz/infrared IR signal  and a MQTT broker
+   Send and receiving command by MQTT
+   
+   This is the Climate Addon:
+   - Measures Temperature, Humidity and Pressure
+   - Generates Values for: Temperature in degrees C and F, Humidity in %, Pressure in Pa, Altitude in Meter and Feet
+   - Required Hardware Module: HTU21
+   - Required Library: SparkFun HTU21 Library v1.1.3
+
+   Connection Schemata:
+   --------------------
+
+   HTU21 ------> Arduino Uno ----------> ESP8266
+   ==============================================
+   Vcc ---------> 5V -------------------> Vu (5V)
+   GND ---------> GND ------------------> GND
+   SCL ---------> Pin A5 ---------------> D1
+   SDA ---------> Pin A4 ---------------> D2
+ 
+    Copyright: (c) Hans-Juergen Dinges
+  
+    This file is part of OpenMQTTGateway.
+    
+    OpenMQTTGateway is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    OpenMQTTGateway is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "User_config.h"
+
+#ifdef ZsensorHTU21
+#include "Wire.h" // Library for communication with I2C / TWI devices
+#include <stdint.h>
+#include "SparkFunHTU21D.h"
+#include "config_HTU21.h"
+
+//Global sensor object
+HTU21D mySensor;
+
+void setupZsensorHTU21()
+{
+  delay(10); // Gives the Sensor enough time to turn on
+  Serial.println("HTU21 Initialized - begin()");
```

Comment:
May you use Log.notice instead of Serial.println, you can find examples on the others sensor implementations

Reply:
My mistake. Changed to Log.notice

Reply:
Thanks